### ISSUE TITLE
Import policy from json file

### ISSUE THREAD

**Issue by vibl** [kopia/kopia#2785]
From what I read in the issues and forum, it doesn't seem possible to import a policy from an exported json file.

Can you confirm that?

If so, is it a planned feature?

If not, could it be?

It does seem a basic feature to me for streamlining backup management in multiple machines, but maybe I'm missing something.

**Comment by DerivativeOfLog7** [comment]
Yeah, the lack of a way to import a preexisting policy seems a big omission to me too

**Comment by taltamir** [comment]
oddly enough, the docs _claim_ there is a policy export feature.
https://kopia.io/docs/reference/command-line/common/policy-export/

but this command does not actually exist.
`kopia: error: expected command but got "export", try --help`
the closest sub-command is `policy show`

```
$ kopia policy --help

usage: kopia policy <command> [<args> ...]
Commands to manipulate snapshotting policies.

Flags:
      --[no-]help             Show context-sensitive help (also try --help-long
                              and --help-man).
      --[no-]version          Show application version.
      --log-file=LOG-FILE     Override log file.
      --log-dir="/home/t/.cache/kopia"  
                              Directory where log files should be written.
                              ($KOPIA_LOG_DIR)
      --log-level=info        Console log level
      --file-log-level=debug  File log level
      --[no-]help-full        Show help for all commands, including hidden
      --config-file="repository.config"  
                              Specify the config file to use
                              ($KOPIA_CONFIG_PATH)
  -p, --password=PASSWORD     Repository password. ($KOPIA_PASSWORD)
      --[no-]persist-credentials  
                              Persist credentials
                              ($KOPIA_PERSIST_CREDENTIALS_ON_CONNECT)
      --[no-]use-keyring      Use Gnome Keyring for storing repository password.
                              ($KOPIA_USE_KEYRING)

Subcommands:
  policy
    edit [<flags>] [<target>...]
    list [<flags>]
    delete [<flags>] [<target>...]
    set [<flags>] [<target>...]
    show [<flags>] [<target>...]
```

**Comment by derivativeoflog7** [comment]
> oddly enough, the docs _claim_ there is a policy export feature. https://kopia.io/docs/reference/command-line/common/policy-export/ `$ kopia policy export [target]` but this command does not actually exist. `kopia: error: expected command but got "export", try --help`
> 
> the closest sub-command is `policy show`
> 
> ```
> $ kopia policy --help
> 
> usage: kopia policy <command> [<args> ...]
> Commands to manipulate snapshotting policies.
> 
> Flags:
>       --[no-]help             Show context-sensitive help (also try --help-long
>                               and --help-man).
>       --[no-]version          Show application version.
>       --log-file=LOG-FILE     Override log file.
>       --log-dir="/home/t/.cache/kopia"  
>                               Directory where log files should be written.
>                               ($KOPIA_LOG_DIR)
>       --log-level=info        Console log level
>       --file-log-level=debug  File log level
>       --[no-]help-full        Show help for all commands, including hidden
>       --config-file="repository.config"  
>                               Specify the config file to use
>                               ($KOPIA_CONFIG_PATH)
>   -p, --password=PASSWORD     Repository password. ($KOPIA_PASSWORD)
>       --[no-]persist-credentials  
>                               Persist credentials
>                               ($KOPIA_PERSIST_CREDENTIALS_ON_CONNECT)
>       --[no-]use-keyring      Use Gnome Keyring for storing repository password.
>                               ($KOPIA_USE_KEYRING)
> 
> Subcommands:
>   policy
>     edit [<flags>] [<target>...]
>     list [<flags>]
>     delete [<flags>] [<target>...]
>     set [<flags>] [<target>...]
>     show [<flags>] [<target>...]
> ```

Added in 0.18!
This issue can probably be closed now

**Comment by julio-lopez** [comment]
Addressed #4020
